osu.Game/Screens/OnlinePlay/Lounge/Components/DrawableRoomParticipantsList.cs

```diff
@@ -194,6 +257,16 @@ private void updateHiddenUsers()
             }
         }
 
+        private void onHostChanged(ValueChangedEvent<User> host)
+        {
+            hostAvatar.User = host.NewValue;
+            hostText.Clear();
+
+            hostText.AddText(@"hosted by ");
+            if (host.NewValue != null)
+                hostText.AddUserLink(host.NewValue);
```

Comment:
is new host being `null` viable? i guess it may be once the last user from a room leaves - but if it is going to be `null`, should the "hosted by" text just be hanging there like that, rather than just leaving blank?

(also as a nit, the `hosted by` string should probably not be using the raw prefix)

Reply:
`null` is the default state of `Room.Host`, so technically yes but in practice probably not. It's much of a muchness whether the `hosted by` text remains or not imo.